### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by Radiovolt** [DarkflameUniverse/DarkflameServer#448]
### Make sure you've done the following:

- [X] I have checked that this issue has not already been reported.

- [X] I have validated that this issue is not a syntax error of either MySQL or SQLite.

- [X] I have pulled the latest version of the main branch of DarkflameServer and have confirmed that the issue exists there.

### Issue Description

![2022-02-08 (5)](https://user-images.githubusercontent.com/80007565/152970962-3ee1fa60-ea63-4884-a7a6-c59ad9e707e5.png)
Daily Mission for Cannon Cove Shoting Galery not work.

### Reproduction steps

I Accept this daily Mission from Venture League Vendor on Nimbus Plaza. 

### Expected Behavior

After accepting the task and starting the shooting gallery mini-game, points in the task are not counted.

### Environment

Server runing on Windowsvia WSL 1

**Comment by EmosewaMC** [comment]
As of right now this is a client side issue.  Don't believe a fix is possible.

**Comment by Radiovolt** [comment]
I see. I thought the problem was on the server side. But, I think, let it be open for now, for information that there is such a problem.

**Comment by averysumner** [comment]
> As of right now this is a client side issue. Don't believe a fix is possible at this time. For some reason the client will not allow any cannon cove missions that have the target `performact_score`. This is also true for the 25k, 75k and 150k missions. The only reason the 25k one works is because there is a special case to force that mission to be complete.

If it worked in live, this is definitely an issue on our end…

**Comment by Jettford** [comment]
Is there a need for another script patch maybe

**Comment by EmosewaMC** [comment]
> > As of right now this is a client side issue. Don't believe a fix is possible at this time. For some reason the client will not allow any cannon cove missions that have the target `performact_score`. This is also true for the 25k, 75k and 150k missions. The only reason the 25k one works is because there is a special case to force that mission to be complete.
> 
> If it worked in live, this is definitely an issue on our end…

Here's my reasoning for this assumption.
https://github.com/DarkflameUniverse/DarkflameServer/blob/main/dScripts/SGCannon.cpp#L566-L584
This code snippet said "For some reason the client thinks this mission is not complete?".  I went ahead and deleted that and tested the mission 229 again and lo and behold the mission did not complete.  In the database, the charxml shows the current value for mission 229 to be set to 25000 and its mission state is 4 (Mission state ready to complete.)  This issue is likely either far more complex, or its just a client issue.  I am leaning on the latter because literally no other mission has an issue with completing except this one.

**Comment by averysumner** [comment]
> Is there a need for another script patch maybe

Nope, script patches shouldn’t be needed at all. The only reason we added the AGS one is because we couldn’t figure it out and it was a critical issue. Supposedly @lcdr has a solution to this.
